### ISSUE TITLE
If you don't pass a value to `renderHeader`, the tabs don't show up

### ISSUE THREAD

**Issue by alexpchin** [PedroBern/react-native-collapsible-tab-view#39]
I have a couple of pages that require tabs that do not require a header. I thought that the default of `renderHeader` was `() => null` so passing no header would still display the tabs as if we were using `react-native-tab-view`.

```ts
<CollapsibleTabView<Route>
      navigationState={{ index, routes }}
      renderScene={renderScene}
      onIndexChange={handleIndexChange}
      // renderHeader={renderHeader}
      headerHeight={HEADER_HEIGHT}
      {...props}
    />
```

However, if you omit `renderHeader`, the tabs do not show.

![Screenshot 2021-01-13 at 16 13 19](https://user-images.githubusercontent.com/40461/104478908-bd1cee80-55ba-11eb-958b-8aa7d5da06b9.png)

**Comment by PedroBern** [comment]
This bug was probably introduced at some point. This didn't use to happen. But I think there is no point in using it without the `renderHeader`, a better option is to use the regular tab view.

**Comment by PedroBern** [comment]
It is probably easy to fix, I will try to see it later. Or you can open a PR :)

**Comment by alexpchin** [comment]
It's useful to have if you want to conditionally render a header. I think it has something to do with this PR?

https://github.com/PedroBern/react-native-collapsible-tab-view/pull/18/files

**Comment by PedroBern** [comment]
yes you are right, probably [here](https://github.com/PedroBern/react-native-collapsible-tab-view/pull/18/files#diff-df3446ac566c5f3c620a18f4fd9951b40e3f6e703393f8eabdf417cbbd5ce0d5L276). 

Not sure, but maybe this will fix:

```diff
--- {renderHeader()}
+++ {renderHeader && renderHeader()}
```

**Comment by alexpchin** [comment]
Hmmm... I don't think so... I think it's something to do with the value of `translateY` on the `Animated.View`

**Comment by alexpchin** [comment]
https://github.com/PedroBern/react-native-collapsible-tab-view/pull/40

**Comment by PedroBern** [comment]
@alexpchin released in v2.0.1 👍  thanks
